src/sbp_fileio.c

```diff
@@ -81,16 +81,16 @@ static void read_cb(u16 sender_id, u8 len, u8 msg[], void* context)
   len += cfs_read(f, buf + len, readlen);
   cfs_close(f);
 
-  sbp_send_msg(SBP_MSG_FILEIO_READ, len, buf);
+  sbp_send_msg(SBP_MSG_FILEIO_READ_RESPONSE, len, buf);
```

Comment:
`len` is an 8-bit value, so you can't overrun the buffer - it'll just wrap. If you try and in a check for this, the compiler is warning:

```
sbp_fileio.c:86:3: error: comparison is always false due to limited range of data type
```